config/rbac/role.yaml

```diff
@@ -6,6 +6,44 @@ metadata:
   creationTimestamp: null
   name: manager-role
 rules:
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  - configmaps/status
+  - events
+  - namespaces
+  - namespaces/status
+  - secrets
+  - secrets/status
+  - services
+  - services/status
```

Comment:
Why do we need this?
What are the configmaps, secrets, namespaces, etc. that we need to read?